### PR TITLE
Updated LinkButton & ActionButton styling

### PR DESCRIPTION
**Pull request by krischarbonneau**
### Description

This PR addresses the focus, active & hover styling for both the `LinkButton` and `ActiveButton` components. I've adjusted the colors on the buttons to match what is on [Canada.ca](https://www.canada.ca) and also swapped out the `Link` components on the index page with our `LinkButton` component.

As seen below, the active styling passes WCAG AA standards:
Screenshot 2022-10-06 101124

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Use your contrast checker extension of choice (I use [WCAG Contrast Color Checker](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf?hl=en)) and test the contrast of the focus/hover/active states by opening the developer console, selecting the element and forcing element state under the styles tab.

### CODE REVIEW

File: components/LinkButton.tsx [hunk 1]
```diff
@@ -4,14 +4,19 @@ import Link from 'next/link'
 export interface LinkButton {
```
Comment (sebastien-comeau) [inline components/LinkButton.tsx:4]: Rename the interface to `LinkButtonProps`

Comment (sebastien-comeau) [reply components/LinkButton.tsx:4]: LinkButtonProps should be this, imo:

```ts
export interface LinkButtonProps {
  children: ReactNode
  href?: string
  id?: string
  lang?: string
}
```

File: components/LinkButton.tsx [hunk 2]
```diff
@@ -4,14 +4,19 @@ import Link from 'next/link'
...
   href: string
   text: string
+  id?: string
+  lang?: string
 }
 
-const LinkButton: FC<LinkButton> = ({ href, text }) => {
+const LinkButton: FC<LinkButton> = ({ href, text, id, lang }) => {
   return (
     <Link href={href} passHref>
```
[the remark this answers is not in the document]
Comment (s-laugh) [reply components/LinkButton.tsx:13]: I disagree with this; almost every time we should use next/link even when it's not needed. This way helps reduce duplication of importing it everywhere (and potentially forgetting it somewhere)

**Review by s-laugh — APPROVED** [review]
I'm good with these changes as is.

File: components/LinkButton.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (krischarbonneau) [reply components/LinkButton.tsx:13]: Leaving this out of this PR as this is only to address the styling of the buttons so they meet WCAG color contrast standards, we can have more discussion about your proposed changes at standup tomorrow.

**Pull request merged**